Sprint_Tests/vova_timonin.md

```diff
@@ -0,0 +1,23 @@
+1. Раньше он хорошо разбирался в верстке и работе на передовых технологиях на фронте. После небольшого перерыва в несколько лет и глубокой депрессии плюс прослушивания тяжелой музыки, он слабо разбирается в современных тенденциях и много пьет. Если бы он справился с депрессией после смерти жены и ребенка, у него бы сейчас не стоял вопрос поиска работы с релевантной зарплатой.
+
+Previously, he was good in web design and advanced technologies of the front-end. After a short break for several years and deep depression plus hard music listening he is weak in modern trends and drinks a lot. If he had coped with depression after deathes of his wife and child he wouldn't have a question of search a relevant salary job.    
+
+2. Будь у него цель и адекватные зарплатные ожидания он бы уже работал на удаленке где нибудь в штатах или на Бали с перспективой перебраться на Канарские острова и оформить испанское гражданство. Он с детства мечтал жениться на испанке и рано выйти на пенсию и разводить виноград на побережье Балеарского моря.
+
+If he had a target and adequate salary expectations he would already have a remote work anywhere in USA or Bali with perspective of a relocation to Canary and obtaining Spanish citizenship. Since childhood he dreamed of marrying a Spanish girl and early retirenment and growing grapes on the coast of Balearic Sea.
+
+3. Если не знаешь как работает кусок кода, написанный коллегой тебе придется покрыть его консоль логами и проследить всю логическую цепочку. Это трудоемкое занятие. Если бы у нас была документация к этой фиче или хотя бы тикет на реализацию фичи с описанием бизнес логики, мы бы вылили фикс в течении пары часов.
+
+If you don't know how the part of code written by your colleague works, you will have to cover it with console logs and trace the logic chain. It is a hard work. If we had documentation of this feature or at least the ticket for the implementation of the feature with business logic description, we would deploy a fix for two hours. 
+
+4. Я хочу чтобы вы не стеснялись выражать свои мысли на иностранном языке не взирая на ошибки, которые могут появиться ввиду отсутствия практики и нерегулярности занятий языком. Я хочу чтобы каждый из вас до конца года прочел по книге в оригинале, язык оригинала естественно английский, и получил удовольствие от преодоления языкового барьера в общении.
+
+I would like you don't hasitate to talk about your thoughts in a foreign language in spite of mistakes that can be appeared because of no practice and irregular language learning. I would like each of you will read a book in origin until the end of the year, of course the original language is English, and enjoy overcoming language barrier in communication. 
```

Comment:
I want you not to hesitate to express your ...
that can appear
lack of practice
I would like each of you to read 
in the original
till the end of the year.